pages/_en-US/twilightmenu/custom-dsi-3ds-skins.md

```diff
@@ -221,10 +221,10 @@ You may configure various options on how the theme is drawn in the `theme.ini` t
 You may add specific override options to `theme.ini` for use in Macro Mode. To do this, add `[MACRO]` to a blank line at the bottom of the configuration file, then add any specified configurations below it.
 
 ## Custom background music and sound effects
-The DSi and 3DS themes also support custom music. See [DSi/3DS skins - Custom SFX](custom-dsi-3ds-sfx) for more details.
+Skins can [set an override](custom-dsi-3ds-sfx) to custom music and sound effects for the Nintendo DSi & Nintendo 3DS themes.
 
 ## Custom fonts
-You may put [Custom Fonts](custom-fonts) in the `font` folder for use in the skin. You can also add override fonts for the date & time using `date_time.nftr`, and the console username with `username.nftr`.
+Custom fonts can be loaded from skins, whether globally or specific sections (`date_time.nftr`/`username.nftr`). Place them in `ui` -> `font`, according to the selection specified in the "[Custom Fonts](custom-fonts)" page.
```

Comment:
I prefer the old version of this paragraph. Also they don't go in a `ui` folder, just `font` in the skin folder.

Reply:
Huh, I forgot where I read that you should use the ui folder. I'll do this as a manual commit.